<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CD Kiosk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelGlobalTitle">
        <property name="styleSheet">
         <string notr="true">font-size: 30pt;</string>
        </property>
        <property name="text">
         <string>CD Kiosk</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="push1Choose">
        <property name="styleSheet">
         <string notr="true">f</string>
        </property>
        <property name="text">
         <string>1 Choose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push2Cart">
        <property name="styleSheet">
         <string notr="true">f</string>
        </property>
        <property name="text">
         <string>2 Cart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push3Burn">
        <property name="styleSheet">
         <string notr="true">f</string>
        </property>
        <property name="text">
         <string>3 Burn</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="centralStack">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="choose">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>-1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Months</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listMonths"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Persons</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listPersons"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Compilations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listCompilations"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Speeches</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listSpeeches"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelAmount">
            <property name="text">
             <string>%0 Pcs.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolAddAmount">
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/gui/icons/add.png</normaloff>:/gui/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolRemoveAmount">
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/gui/icons/remove.png</normaloff>:/gui/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkMP3">
            <property name="text">
             <string>MP3</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushAddToCart">
            <property name="text">
             <string>Add to Cart</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushProceedToReview">
            <property name="styleSheet">
             <string notr="true">font-weight: 800;</string>
            </property>
            <property name="text">
             <string>Proceed</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="review">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-size: 20pt;</string>
          </property>
          <property name="text">
           <string>%0 Items in Cart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="listCart">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushBackToChoose">
            <property name="styleSheet">
             <string notr="true">font-weight: 800;</string>
            </property>
            <property name="text">
             <string>Add more</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/gui/icons/add.png</normaloff>:/gui/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushRemoveFromCart">
            <property name="text">
             <string>Remove Selected</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/gui/icons/remove.png</normaloff>:/gui/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushProceedToBurn">
            <property name="styleSheet">
             <string notr="true">font-weight: 800;</string>
            </property>
            <property name="text">
             <string>Burn</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/gui/icons/burn.png</normaloff>:/gui/icons/burn.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="burn">
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>push1Choose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showChoosePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>963</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push2Cart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCartPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1085</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>1004</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push3Burn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showBurnPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1170</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>1110</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushProceedToBurn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showBurnPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBackToChoose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showChoosePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushProceedToReview</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCartPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1165</x>
     <y>756</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listMonths</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSpeechList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>783</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listPersons</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSpeechList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listCompilations</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSpeechList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>786</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showChoosePage()</slot>
  <slot>showCartPage()</slot>
  <slot>showBurnPage()</slot>
  <slot>updateChooseLists()</slot>
  <slot>updateSpeechList()</slot>
 </slots>
</ui>
